bindgen/ir/location/LocationManager.cpp

```diff
@@ -0,0 +1,17 @@
+#include "LocationManager.h"
+#include "SourceLocation.h"
+
+LocationManager::LocationManager(std::string mainHeaderPath)
+    : mainHeaderPath(std::move(mainHeaderPath)) {}
+
+void LocationManager::loadConfig(const std::string &path) {
+    // TODO: implement
+}
+
+std::shared_ptr<Location>
+LocationManager::getLocation(std::string pathToHeader,
+                             std::string includeLocation) {
```

Comment:
What's the purpose of `includeLocation`?

Reply:
`pathToHeader` minus `includeLocation` will give us relative path to header (the path that is used in `include` for example `bits/stdio.h`).
stdHeader file should contain these relative paths (not only header names) so we will be able to correctly map headers to existing bindings.

Reply:
Oh no `includeLocation` is not an include path